buildroot-external/rootfs-overlay/usr/sbin/hassos-supervisor

```diff
@@ -27,7 +27,10 @@ if [ -n "${SUPERVISOR_CONTAINER_ID}" ]; then
         STOP=$(date --date="${SUPERVISOR_STOP}" +%s)
         SUPERVISOR_RUNTIME=$((STOP-START))
         # Minimial runtime not met, remove container and image
-        if [ "${SUPERVISOR_RUNTIME}" -lt "${SUPERVISOR_MIN_RUNTIME}" ]; then
+        # if there is no update process ongoing
+        if [ "${SUPERVISOR_IMAGE_ID}" != "${SUPERVISOR_CONTAINER_ID}" ]; then
```

Comment:
This check does not belong here, it should be part of the if check that is done at line 21